simple_video_player.html

```diff
@@ -22,6 +22,9 @@
   <input id=volume type=range value=0.8 min=0 max=1.0 step=0.01></input>
   Audio buffer health:
   <progress min=0 max=100></progress>
+  Use AudioContext.outputLatency:
+  <input id=outputLatency type=checkbox></input>
+  (Plug Bluetooth headset for best results)
```

Comment:
Nit: you certainly don't want to plug your Bluetooth headset in, because that defeats the purpose. You want to use it, unplugged.

Reply:
Displaying the offset in use would be nice as well, I suppose. It is customary to display audio latencies in milliseconds or frames, despite the clock being in seconds.

Reply:
Thanks for reviewing @padenot! I've addressed your feedback at https://github.com/chcunningham/wc-talk/pull/7/commits/48fd00c1bddb3cd9b48a5e207ba8fcbfc51e5784